src/components/Moment.js

```diff
@@ -3,14 +3,11 @@ import PropTypes from 'prop-types';
 import moment from 'moment';
 import { FormattedDate } from 'react-intl';
 
-const formatDate = value => new Date(value).toISOString();
-
 const Moment = ({ value, relative }) => {
-  const formattedLongDateStr = moment(value).format('LLLL');
+  const date = new Date(value);
+  const formattedLongDateStr = moment(date).format('LLLL');
```

Comment:
I don't know the cost of `moment(date)` but it would be nice to extract it to a new variable as we potentially build it two times if the date is relative

Reply:
Yes, why not do that while we're there.